pkg/reconciler/instances/istio/actions/performer.go

```diff
@@ -219,6 +233,19 @@ func (c *DefaultIstioPerformer) PatchMutatingWebhook(context context.Context, ku
 	return nil
 }
 
+func (c *DefaultIstioPerformer) selectWebhookToPatch(context context.Context, candidatesNames []string, clientSet clientgo.Interface) (*v1.MutatingWebhookConfiguration, error) {
+	var wh *v1.MutatingWebhookConfiguration
+	var err error
+	for _, webhookName := range candidatesNames {
+		wh, err = clientSet.AdmissionregistrationV1().MutatingWebhookConfigurations().Get(context, webhookName, metav1.GetOptions{})
+		if err != nil {
+			continue
+		}
+		return wh, err
```

Comment:
After you give names to return parameters you can simply `return` from the function and they will be returned automatically